manta-pay/Cargo.toml

```diff
@@ -142,4 +145,4 @@ ws_stream_wasm = { version = "0.7.3", optional = true, default-features = false
 
 [dev-dependencies]
 manta-crypto = { path = "../manta-crypto", default-features = false, features = ["getrandom"] }
-manta-pay = { path = ".", default-features = false, features = ["groth16", "scale", "scale-std", "std", "test"] }
+manta-pay = { path = ".", default-features = false, features = ["download", "groth16", "scale", "scale-std", "std", "test"] }
```

Comment:
Why not just import `manta-parameters` with `download` enabled as a dev-dependency? No need to add a new feature.